.cicd/defaults.json

```diff
@@ -1,6 +1,6 @@
 {
    "leap-dev":{
-      "target":"hotstuff_integration",
+      "target":"controller-self",
```

Comment:
Will this be reverted back?

Reply:
Yes, had to do that for this to compile.